StoreApp/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using StoreApp.Data;
+using StoreApp.Models;
+using System.Linq;
+
+namespace StoreApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var dbContext = new DbContext();
+            dbContext.InitData();
+
+            Console.WriteLine("\n*****Task 1*****\n");
+
+            var productList1 = dbContext.Products.Select(x => x).OrderBy(x => x.Name);
```

Comment:
please move each task to separate method
method should contain only logic
UI elements such as "Console.WriteLine..." will be in Main